fakturogen-comarch-connector/src/main/java/pl/fakturogen/comarch/connector/dto/ComarchAddressDTO.java

```diff
@@ -10,19 +10,13 @@
 @Builder
 @NoArgsConstructor
 @AllArgsConstructor
-public class AddressComarchDTO {
-
-    private Long id;
-
-    private Long idExternalApi;
+public class ComarchAddressDTO {
 
+    private String $id;
```

Comment:
I think that this $id we will not need in DTO, as it will be probably not used anywhere.